cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -1137,3 +1138,25 @@ def test_mixed_security_group_overwrite_validator(head_node_security_groups, que
     )
     expected_message = "make sure.*cluster nodes are reachable" if expect_warning else None
     assert_failure_messages(actual_failures, expected_message)
+
+
+@pytest.mark.parametrize(
+    "root_volume_size, expected_message, ami_size",
```

Comment:
Minor: I think is better to put parameters in the following order: inputs followe by the expected outcome.
So, in this case: `root_volume_size, ami_size, expected_message`

Reply:
Done